crates/sargon-uniffi/src/profile/mfa/security_structures/security_shield_builder.rs

```diff
@@ -18,17 +19,18 @@ use crate::prelude::*;
 /// A builder of `SecurityStructureOfFactorSourceIds` a.k.a. `SecurityShield`,
 /// which contains a MatrixOfFactorSourceIds - with primary, recovery, and
 /// confirmation roles.
-#[derive(Debug, uniffi::Object)]
+#[derive(Debug, Hash, PartialEq, Clone, uniffi::Object)]
+#[uniffi::export(Hash, Eq)]
 pub struct SecurityShieldBuilder {
-    wrapped: RwLock<sargon::SecurityShieldBuilder>,
+    wrapped: Arc<sargon::SecurityShieldBuilder>,
```

Comment:
Hmm weird that Git think this is new... I've merged main into my branch. hmm....